packages/store-sync/src/getRecord.bench.ts

```diff
@@ -0,0 +1,31 @@
+import { bench, describe } from "vitest";
+import { getComponentEntities, getComponentValue } from "@latticexyz/recs";
+import { eq } from "drizzle-orm";
+import { mudStoreTables } from "./sqlite";
+import { blocks } from "../test/blocks";
+import {
+  components,
+  db,
+  recsStorageAdapter,
+  sqliteStorageAdapter,
+  tables,
+  useStore,
+  zustandStorageAdapter,
+} from "../test/utils";
+import { singletonEntity } from "./recs";
+
+for (const block of blocks) {
+  await recsStorageAdapter(block);
+  await zustandStorageAdapter(block);
+  await sqliteStorageAdapter(block);
+}
```

Comment:
Not blocking since the impact is minimal but probably faster to do these in parallel. Could even shorten this to
```ts
await Promise.all(blocks.flatMap((block) => [
  recsStorageAdapter(block),
  zustandStorageAdapter(block),
  sqliteStorageAdapter(block)
]));
```

Reply:
wait wait, I was wrong on this - storage adapters are meant to run each block serially, waiting for the previous block's storage adapter call to complete before running the next

so it'd look like
```ts
for (const block of blocks) {
  await Promise.all([recsStorageAdapter(block), zustandStorageAdapter(block), sqliteStorageAdapter(block)]);
}
```

Reply:
Fixed!